### ISSUE TITLE
Switch court

### ISSUE THREAD

**Issue by mchiapello** [openvolley/ovscout2#20]
Hello,
new season started! I tried to scout the first match today, but the "switch court" button is not working properly.

Here the default:
<img width="331" alt="image" src="https://github.com/openvolley/ovscout2/assets/18676022/70f4f330-396b-42a1-ae05-9f6795b47927">

Here after me pressing the (double arrow) "switch court" button:
<img width="331" alt="image" src="https://github.com/openvolley/ovscout2/assets/18676022/177a4c41-67a9-44e9-b590-cccd6dfb6b97">

As you can see the teams are not changing court

**Comment by mchiapello** [comment]
I tried to reproduced the error using the demo file, but:

```
> library(ovscout2)
Loading required package: datavolley
> ov_scouter_demo()
Error in ov_scouter_demo() : could not find function "ov_scouter_demo"
```

**Comment by raymondben** [comment]
Someone else noticed the switch court problem recently but we could not reproduce it to be able to troubleshoot. If you can reproduce, can you send me the .ovs file please?

**Comment by mchiapello** [comment]
[BCV-GianFer.ovs.zip](https://github.com/openvolley/ovscout2/files/13333499/BCV-GianFer.ovs.zip)

**Comment by raymondben** [comment]
Have made a quick edit, I think it fixes it (v0.2.7, https://github.com/openvolley/ovscout2/commit/43f8835f68ef28a5ccb12decae9cd8b0d29f153a).

**Comment by mchiapello** [comment]
It works!
Thanks
